api/v1alpha1/physical_machine_chaos_webhook_test.go

```diff
@@ -60,7 +64,12 @@ var _ = Describe("physicalmachinechaos_webhook", func() {
 							PhysicalMachineSelector: PhysicalMachineSelector{
 								Address: []string{""},
 							},
-							ExpInfo: ExpInfo{},
+							ExpInfo: ExpInfo{
+								StressCPU: &StressCPUSpec{
+									Load:    10,
+									Workers: 1,
+								},
+							},
```

Comment:
How about adding more test cases here? (for example, tests for `size`, `percent` field)

Reply:
OK, I will add them 👍

Reply:
Done, PTAL again